.travis.yml

```diff
@@ -10,20 +10,20 @@ env:
   global:
   - DOCKER_REPO=$TRAVIS_REPO_SLUG
 before_install:
-- apt-cache madison docker-engine
-- travis_retry sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-engine
+- apt-cache madison docker-ce
```

Comment:
These shouldn't appear in your diff.  I feel like your rebase was messed up somehow.  Are you sure you rebased your changes on top of the HEAD of the master branch?